tests/helpers.py

```diff
@@ -66,6 +66,10 @@ def setup_method(self, method):
         self.client = self.app.test_client()
         self.get_user_patch = None
 
+        self.briefs_dashboard_url = "/buyers"
+        if self.app.config.get('FEATURE_FLAGS_DIRECT_AWARD_PROJECTS'):
+            self.briefs_dashboard_url = "/buyers/requirements/digital-outcomes-and-specialists"
+
     def teardown_method(self, method):
```

Comment:
This doesn't live on the main tests base class if it's only ever used in `test_buyers.py`. We want to keep base classes as slim as possible and attach attributes like these to classes specific to the tests that use them.

This would also be more pythonic as a ternary assignment.
http://book.pythontips.com/en/latest/ternary_operators.html

Reply:
il get this moved.

Reply:
Oh wait, there was a reason why it had to be there. For some reason the self.apps couldn't see the config attribute inside of the subclasses. I will look to reimplement it and see if it works.